King-Benx/WeConnect-Android-Java
# Adding retrofit dependencies

#### What does this PR do?
This PR delivers dependencies responsible for making the API calls using Retrofit

#### Description of Task to be completed?
In order to be able to communicate with the WeConnect API, we need the retrofit library

#### How should this be manually tested?
1. Clone the repository onto your machine and build the app. 
2. Run ./gradlew build
3. You can also check the app/build.gradle and you will notice that retrofit dependencies exist.

#### Background context
NONE
